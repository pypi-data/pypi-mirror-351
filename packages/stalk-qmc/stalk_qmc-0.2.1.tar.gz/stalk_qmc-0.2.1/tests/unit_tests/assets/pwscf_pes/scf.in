&CONTROL
   calculation     = 'scf'
   outdir          = 'pwscf_output'
   prefix          = 'pwscf'
   pseudo_dir      = './'
/

&SYSTEM
   degauss         = 0.0001
   ecutrho         = 600
   ecutwfc         = 300
   ibrav           = 0
   input_dft       = 'pbe'
   nat             = 2
   nosym           = .false.
   nspin           = 1
   ntyp            = 1
   occupations     = 'smearing'
   smearing        = 'gaussian'
   tot_charge      = 0
/

&ELECTRONS
   conv_thr        = 1e-09
   electron_maxstep = 200
   mixing_beta     = 0.7
/


ATOMIC_SPECIES 
   C  12.011 C.ccECP.upf

ATOMIC_POSITIONS bohr
   C        0.00000000       0.00000000       0.00000000 
   C        1.68700061       1.68700061       1.68700061 

K_POINTS automatic
   8 8 8  0 0 0 

CELL_PARAMETERS bohr
         3.37400122       3.37400122       0.00000000 
        -0.00000000       3.37400122       3.37400122 
         3.37400122      -0.00000000       3.37400122 


