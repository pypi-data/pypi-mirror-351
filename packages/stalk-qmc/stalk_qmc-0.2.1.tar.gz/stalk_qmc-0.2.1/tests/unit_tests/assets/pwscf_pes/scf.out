
     Program PWSCF v.7.4 starts on 15Apr2025 at 10:10:18 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     8 processors

     MPI processes distributed on     1 nodes
     0 MiB available memory on the printing compute node when the environment starts

     Reading input from scf.in

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4
scan_end: No INFO end statement, corrupted file?

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

     Message from routine set_cutoff:
     ecutrho < 4*ecutwfc, are you sure?
     Message from routine setup:
     using ibrav=0 with symmetry is DISCOURAGED, use correct ibrav instead

     R & G space division:  proc/nbgrp/npool/nimage =       8
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         119     119     68                 2398     2398    1036
     Max         120     120     69                 2400     2400    1037
     Sum         955     955    547                19195    19195    8291

     Using Slab Decomposition



     bravais-lattice index     =            0
     lattice parameter (alat)  =       4.7716  a.u.
     unit-cell volume          =      76.8185 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =         8.00
     number of Kohn-Sham states=            8
     kinetic-energy cutoff     =     300.0000  Ry
     charge density cutoff     =     600.0000  Ry
     scf convergence threshold =      1.0E-09
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)

     celldm(1)=   4.771558  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   0.707107   0.707107   0.000000 )  
               a(2) = (  -0.000000   0.707107   0.707107 )  
               a(3) = (   0.707107  -0.000000   0.707107 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  0.707107  0.707107 -0.707107 )  
               b(2) = ( -0.707107  0.707107  0.707107 )  
               b(3) = (  0.707107 -0.707107  0.707107 )  


     PseudoPot. # 1 for C  read from file:
     ./C.ccECP.upf
     MD5 check sum: 9ad03c3cf6429d54972c076c8f6c2b50
     Pseudo is Norm-conserving, Zval =  4.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of 1122 points,  1 beta functions with: 
                l(1) =   0

     atomic species   valence    mass     pseudopotential
     C                 4.00    12.01100     C ( 1.00)

     48 Sym. Ops., with inversion, found (24 have fractional translation)



   Cartesian axes

     site n.     atom                  positions (alat units)
         1        C      tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2        C      tau(   2) = (   0.3535534   0.3535534   0.3535534  )

     number of k points=    29  Gaussian smearing, width (Ry)=  0.0001
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0039062
        k(    2) = (   0.0883883  -0.0883883   0.0883883), wk =   0.0312500
        k(    3) = (   0.1767767  -0.1767767   0.1767767), wk =   0.0312500
        k(    4) = (   0.2651650  -0.2651650   0.2651650), wk =   0.0312500
        k(    5) = (  -0.3535534   0.3535534  -0.3535534), wk =   0.0156250
        k(    6) = (   0.0000000   0.0000000   0.1767767), wk =   0.0234375
        k(    7) = (   0.0883883  -0.0883883   0.2651650), wk =   0.0937500
        k(    8) = (   0.1767767  -0.1767767   0.3535534), wk =   0.0937500
        k(    9) = (  -0.4419417   0.4419417  -0.2651650), wk =   0.0937500
        k(   10) = (  -0.3535534   0.3535534  -0.1767767), wk =   0.0937500
        k(   11) = (  -0.2651650   0.2651650  -0.0883883), wk =   0.0937500
        k(   12) = (  -0.1767767   0.1767767   0.0000000), wk =   0.0468750
        k(   13) = (   0.0000000   0.0000000   0.3535534), wk =   0.0234375
        k(   14) = (   0.0883883  -0.0883883   0.4419417), wk =   0.0937500
        k(   15) = (  -0.5303301   0.5303301  -0.1767767), wk =   0.0937500
        k(   16) = (  -0.4419417   0.4419417  -0.0883883), wk =   0.0937500
        k(   17) = (  -0.3535534   0.3535534   0.0000000), wk =   0.0468750
        k(   18) = (  -0.0000000   0.0000000   0.5303301), wk =   0.0234375
        k(   19) = (  -0.6187184   0.6187184  -0.0883883), wk =   0.0937500
        k(   20) = (  -0.5303301   0.5303301   0.0000000), wk =   0.0468750
        k(   21) = (   0.0000000   0.0000000  -0.7071068), wk =   0.0117188
        k(   22) = (   0.1767767   0.0000000   0.3535534), wk =   0.0937500
        k(   23) = (  -0.4419417   0.6187184  -0.2651650), wk =   0.1875000
        k(   24) = (  -0.3535534   0.5303301  -0.1767767), wk =   0.0937500
        k(   25) = (  -0.5303301   0.7071068  -0.1767767), wk =   0.0937500
        k(   26) = (  -0.4419417   0.6187184  -0.0883883), wk =   0.1875000
        k(   27) = (  -0.3535534   0.5303301   0.0000000), wk =   0.0937500
        k(   28) = (   0.1767767  -0.0000000  -0.7071068), wk =   0.0468750
        k(   29) = (   0.3535534   0.0000000  -0.7071068), wk =   0.0234375

     Dense  grid:    19195 G-vectors     FFT dimensions: (  40,  40,  40)

     Estimated max dynamical RAM per process >       5.10 MB

     Estimated total dynamical RAM >      40.79 MB

     Initial potential from superposition of free atoms

     starting charge       6.0000, renormalised to       8.0000
     Starting wfcs are    8 randomized atomic wfcs

     total cpu time spent up to now is        0.2 secs

     Self-consistent Calculation

     iteration #  1     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  3.4

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     Davidson diagonalization with overlap
     ethr =  1.82E-04,  avg # of iterations =  2.2

     total cpu time spent up to now is        0.7 secs

     total energy              =     -22.74901248 Ry
     estimated scf accuracy    <       0.01481982 Ry

     iteration #  2     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.85E-04,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.9 secs

     total energy              =     -22.74983433 Ry
     estimated scf accuracy    <       0.00138768 Ry

     iteration #  3     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.73E-05,  avg # of iterations =  1.7

     total cpu time spent up to now is        1.1 secs

     total energy              =     -22.74988007 Ry
     estimated scf accuracy    <       0.00000135 Ry

     iteration #  4     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.68E-08,  avg # of iterations =  3.6

     total cpu time spent up to now is        1.4 secs

     total energy              =     -22.74988263 Ry
     estimated scf accuracy    <       0.00000033 Ry

     iteration #  5     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.15E-09,  avg # of iterations =  2.0

     total cpu time spent up to now is        1.7 secs

     total energy              =     -22.74988260 Ry
     estimated scf accuracy    <       0.00000030 Ry

     iteration #  6     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.69E-09,  avg # of iterations =  1.0

     total cpu time spent up to now is        1.8 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 (  6735 PWs)   bands (ev):

    -8.2761  13.3416  13.3416  13.3416  18.9498  18.9498  18.9498  26.9575

          k = 0.0884-0.0884 0.0884 (  6726 PWs)   bands (ev):

    -7.8404  10.9205  12.6878  12.6878  19.8120  19.8120  20.2511  27.2995

          k = 0.1768-0.1768 0.1768 (  6720 PWs)   bands (ev):

    -6.5475   6.7293  11.5924  11.5924  21.1570  21.1570  21.5629  28.1605

          k = 0.2652-0.2652 0.2652 (  6745 PWs)   bands (ev):

    -4.4700   2.6317  10.8090  10.8090  21.7495  21.7495  22.1519  28.9042

          k =-0.3536 0.3536-0.3536 (  6754 PWs)   bands (ev):

    -2.4866  -0.0591  10.5362  10.5362  21.8201  21.8201  22.3156  29.1068

          k = 0.0000 0.0000 0.1768 (  6719 PWs)   bands (ev):

    -7.6953  11.6127  11.6127  12.0127  18.5426  21.1781  21.1781  26.8825

          k = 0.0884-0.0884 0.2652 (  6732 PWs)   bands (ev):

    -6.6883   8.2382  10.4610  11.3055  19.2045  22.7398  22.8045  26.3289

          k = 0.1768-0.1768 0.3536 (  6743 PWs)   bands (ev):

    -4.8646   4.1951   9.8127  10.1361  20.3763  23.4694  23.5128  26.9026

          k =-0.4419 0.4419-0.2652 (  6747 PWs)   bands (ev):

    -2.5314   0.7524   9.1815   9.8114  20.9866  23.1865  23.7413  28.6056

          k =-0.3536 0.3536-0.1768 (  6729 PWs)   bands (ev):

    -3.2977   1.7534   8.7496  10.4278  21.5807  22.5357  23.6356  28.8321

          k =-0.2652 0.2652-0.0884 (  6725 PWs)   bands (ev):

    -5.5585   5.5508   8.9668  11.4991  21.4908  22.0344  23.0207  27.6414

          k =-0.1768 0.1768 0.0000 (  6723 PWs)   bands (ev):

    -7.1188   9.3445  10.0840  12.5831  20.1127  21.4423  21.5593  27.3380

          k = 0.0000 0.0000 0.3536 (  6715 PWs)   bands (ev):

    -5.9732   8.8039   9.2131   9.2131  17.8084  23.6476  24.9876  24.9876

          k = 0.0884-0.0884 0.4419 (  6751 PWs)   bands (ev):

    -4.4324   5.7915   8.1067   8.4825  18.5415  22.7201  26.2963  26.3468

          k =-0.5303 0.5303-0.1768 (  6745 PWs)   bands (ev):

    -2.2158   2.5114   6.5836   8.4576  19.4482  24.8311  26.2112  26.7882

          k =-0.4419 0.4419-0.0884 (  6740 PWs)   bands (ev):

    -1.5218   1.7138   5.5789   9.1316  20.2198  25.1671  26.7446  28.2410

          k =-0.3536 0.3536 0.0000 (  6733 PWs)   bands (ev):

    -3.7772   4.6007   5.7214  10.3244  21.3102  23.3896  25.7789  27.4100

          k =-0.0000 0.0000 0.5303 (  6742 PWs)   bands (ev):

    -3.1834   4.7076   7.5946   7.5946  17.4985  20.0993  28.5647  28.5647

          k =-0.6187 0.6187-0.0884 (  6751 PWs)   bands (ev):

    -1.2097   2.6053   5.9912   7.4120  18.3416  20.3796  29.1017  29.3881

          k =-0.5303 0.5303 0.0000 (  6749 PWs)   bands (ev):

     0.2668   1.5421   4.0230   7.9702  19.0455  24.1399  27.7943  29.1230

          k = 0.0000 0.0000-0.7071 (  6782 PWs)   bands (ev):

     0.5000   0.5000   7.0352   7.0352  18.1254  18.1254  30.2823  30.2823

          k = 0.1768 0.0000 0.3536 (  6740 PWs)   bands (ev):

    -5.4135   6.8049   7.8309  10.4480  19.9443  23.4328  24.9798  25.1851

          k =-0.4419 0.6187-0.2652 (  6737 PWs)   bands (ev):

    -3.3767   3.4575   7.0469   9.2531  21.3399  23.0264  25.8528  26.7586

          k =-0.3536 0.5303-0.1768 (  6758 PWs)   bands (ev):

    -1.4630   0.9457   7.0097   8.6470  21.6602  22.8581  26.5160  28.3105

          k =-0.5303 0.7071-0.1768 (  6740 PWs)   bands (ev):

    -2.6728   4.2768   6.3384   7.3939  19.6963  21.9717  26.4320  28.5718

          k =-0.4419 0.6187-0.0884 (  6749 PWs)   bands (ev):

    -0.4752   2.1146   4.9164   6.9861  20.9031  24.2797  25.9806  28.3958

          k =-0.3536 0.5303 0.0000 (  6740 PWs)   bands (ev):

    -1.2694   2.8772   4.4638   7.8598  21.7416  24.1608  26.5287  27.9966

          k = 0.1768-0.0000-0.7071 (  6746 PWs)   bands (ev):

     0.8356   0.8356   6.0198   6.0198  20.2455  20.2455  29.0256  29.0256

          k = 0.3536 0.0000-0.7071 (  6740 PWs)   bands (ev):

     1.2830   1.2830   5.0153   5.0153  23.9842  23.9842  25.7758  25.7758

     the Fermi energy is    15.8263 ev

!    total energy              =     -22.74988263 Ry
     estimated scf accuracy    <          8.8E-10 Ry
     smearing contrib. (-TS)   =      -0.00000000 Ry
     internal energy E=F+TS    =     -22.74988263 Ry

     The total energy is F=E-TS. E is the sum of the following terms:
     one-electron contribution =       7.98438510 Ry
     hartree contribution      =       1.90736262 Ry
     xc contribution           =      -7.09665775 Ry
     ewald contribution        =     -25.54497260 Ry

     convergence has been achieved in   6 iterations

     Writing all to output data dir pwscf_output/pwscf.save/ :
     XML data file, charge density, pseudopotentials, collected wavefunctions

     init_run     :      0.12s CPU      0.13s WALL (       1 calls)
     electrons    :      1.61s CPU      1.67s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.08s CPU      0.09s WALL (       1 calls)
     potinit      :      0.01s CPU      0.01s WALL (       1 calls)
     hinit0       :      0.02s CPU      0.02s WALL (       1 calls)

     Called by electrons:
     c_bands      :      1.37s CPU      1.42s WALL (       7 calls)
     sum_band     :      0.21s CPU      0.22s WALL (       7 calls)
     v_of_rho     :      0.02s CPU      0.03s WALL (       7 calls)
     mix_rho      :      0.00s CPU      0.00s WALL (       7 calls)

     Called by c_bands:
     init_us_2    :      0.01s CPU      0.01s WALL (     435 calls)
     cegterg      :      1.36s CPU      1.40s WALL (     203 calls)

     Called by *egterg:
     cdiaghg      :      0.03s CPU      0.03s WALL (     606 calls)
     h_psi        :      1.20s CPU      1.23s WALL (     664 calls)
     g_psi        :      0.00s CPU      0.00s WALL (     432 calls)

     Called by h_psi:
     h_psi:calbec :      0.05s CPU      0.05s WALL (     664 calls)
     vloc_psi     :      1.12s CPU      1.14s WALL (     664 calls)
     add_vuspsi   :      0.02s CPU      0.02s WALL (     664 calls)

     General routines
     calbec       :      0.05s CPU      0.05s WALL (     664 calls)
     fft          :      0.02s CPU      0.02s WALL (      70 calls)
     ffts         :      0.00s CPU      0.00s WALL (       7 calls)
     fftw         :      1.27s CPU      1.29s WALL (   10074 calls)

     Parallel routines

     PWSCF        :      1.79s CPU      1.92s WALL


   This run was terminated on:  10:10:20  15Apr2025            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
